src/prebid.js

```diff
@@ -757,6 +761,18 @@ $$PREBID_GLOBAL$$.setS2SConfig = function(options) {
   adaptermanager.setS2SConfig(config);
 };
 
+/**
+ * Get Prebid config options
+ * @param {object} options
+ */
+$$PREBID_GLOBAL$$.getConfig = config.getConfig;
```

Comment:
I was the one who suggested this.  I think it's a small thing but makes Prebid.js more extensible by third-party libraries.  So overall I think it's a small net-gain for a one-liner.